File: 練習問題改/参考程度にVB.NET版/02-変数と関数.vb
Imports System

Module _02
    Sub Main(args As String())
        Dim a As Integer = Integer.Parse(Console.ReadLine())
        Dim b As Integer = Integer.Parse(Console.ReadLine())
        Dim c As Integer = Integer.Parse(Console.ReadLine())

        Dim res As Double = Ave(a, b, c)
        Console.WriteLine(res)
    End Sub

    Function Ave(a As Integer, b As Integer, c As Integer)
        Dim _sum As Integer = a + b + c
        Dim _ave As Double = _sum / 3
        Return _ave
    End Function
End Module

File: 練習問題改/参考程度にC++版/02-変数と関数.cpp
#include <iostream>

using namespace std;

double ave(int a, int b, int c) {
    int _sum = a + b + c;
    double _ave = _sum / 3;
    return _ave;
}

int main() {
    int a, b, c;

    cin >> a;
    cin >> b;
    cin >> c;

    double res = ave(a, b, c);
    cout << res << endl;

    return 0;
}